Examples/Algorithms/MaterialMapping/include/ACTFW/MaterialMapping/MaterialMapping.hpp

```diff
@@ -8,15 +8,15 @@
 
 #pragma once
 
-#include <Acts/Material/SurfaceMaterialMapper.hpp>
-#include <Acts/Material/VolumeMaterialMapper.hpp>
-#include <Acts/Utilities/Logger.hpp>
 #include <climits>
 #include <memory>
 #include <mutex>
 
 #include "ACTFW/Framework/BareAlgorithm.hpp"
 #include "ACTFW/MaterialMapping/IMaterialWriter.hpp"
```

Comment:
I will be happy when this will be renamed into `ActsExamples`, but this we should do coherently.

Reply:
Agreed.